apps/chat/bots.py

```diff
@@ -4,36 +4,24 @@
 from langchain.memory import ConversationBufferMemory
 from pydantic import ValidationError
 
-from apps.chat.conversation import AgentConversation, AssistantConversation, BasicConversation, Conversation
+from apps.chat.conversation import BasicConversation, Conversation
 from apps.chat.exceptions import ChatException
-from apps.chat.models import ChatMessage, ChatMessageType
 from apps.experiments.models import ExperimentSession, SafetyLayer
+from apps.service_providers.llm_service.runnables import create_experiment_runnable
 
 
 def create_conversation(
     prompt_str: str,
     source_material: str,
     llm: BaseChatModel,
-    experiment_session: Optional[ExperimentSession] = None,
 ) -> Conversation:
     try:
-        if experiment_session and experiment_session.experiment.assistant:
-            return AssistantConversation(experiment_session)
-        if experiment_session and experiment_session.experiment.tools_enabled:
-            return AgentConversation(
-                prompt_str=prompt_str,
-                source_material=source_material,
-                memory=ConversationBufferMemory(return_messages=True),
-                llm=llm,
-                experiment_session=experiment_session,
-            )
-        else:
-            return BasicConversation(
-                prompt_str=prompt_str,
-                source_material=source_material,
-                memory=ConversationBufferMemory(return_messages=True),
-                llm=llm,
-            )
+        return BasicConversation(
```

Comment:
Since this will only be used for the safety boy now, I think the memory component of this object is redundant

Reply:
Yup, I didn't want to change too much in one PR.